samples/client/petstore/kotlin-jvm-volley/src/main/java/org/openapitools/client/infrastructure/Serializer.kt

```diff
@@ -0,0 +1,22 @@
+package org.openapitools.client.infrastructure
+
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.OffsetDateTime
+import java.util.UUID
+
+object Serializer {
```

Comment:
@alisters I just noticed that this class is not used, which means that, OffsetDateTimeAdapter, LocalDateAdapter, ByteArrayAdapter are also not used.
Could you please remove the reference to them in the KotlinClientCodegen.java?
You should remove them from the `supportingFiles`

Reply:
The adapter classes are used inside GsonRequest - src/main/resources/kotlin-client/libraries/jvm-volley/request/GsonRequest.mustache

I've removed the Serialzer as a supporting file. In processJVMVolleyLibrary
  addSupportingSerializerAdapters(infrastructureFolder);
        supportingFiles.remove(new SupportingFile("jvm-common/infrastructure/Serializer.kt.mustache", infrastructureFolder, "Serializer.kt"));


The serializer class is part of jvm-common - src/main/resources/kotlin-client/jvm-common/infrastructure/Serializer.kt.mustache. So we left it in the supporting files that are generated when gson is used as the serializer, even though our library uses gson, but doesn't use it. I guess we avoided another conditional in kotlin code gen for volley AND gson - it's added addSupportingSerializerAdapters, which only knows about serializers at the moment. 

Let me know if https://github.com/OpenAPITools/openapi-generator/pull/10253/commits/d05a42b53eaf6f0348e8e23ccf8c09d69d9e1805  is ok